### PR TITLE
Add festive season references

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

- Added a snowflake to the "TALK TO SALES" button in the main nav 
- Added a candy cane texture as a the border bottom of the main nav
- Added a christmas hat to the "BUILD YOUR FIRST APPLICATION" button in the home page
- Added a snow falling effect to the home page (only on desktop)

I'll revert these changes on January 8th

## Related Issue(s)

[](https://github.com/FlowFuse/customer/issues/104)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_includes/snow-js.njk
Comment (Yndira-FlowForge) [inline src/_includes/snow-js.njk:1]: It's a [gist](https://gist.github.com/anonymous/c39fedf4e4eb5dbdfbcb595677d67ad1), I tried to embed it but the code was being printed, that's why I added it as a file. Let me know if this is a problem and I'll remove it.

**Comment by ZJvandeWeg** [comment]
I'm not sure about the snow, it seems a bit too much. I know I was leading the idea in the meeting, but seeing it play out I'm not too sure anymore. What's your take @Yndira-FlowForge

**Comment by Yndira-FlowForge** [comment]
> What's your take @Yndira-FlowForge

I think is nice touch, but I get why it might seem too much for some people.
I just changed it so the snow only shows in the hero section, it generates fewer flakes and reduced the velocity. I think is more balanced now.

**Comment by ZJvandeWeg** [comment]
Very cool @Yndira-FlowForge !

**Pull request merged**